src/widgets/forms/forms-utils.ts

```diff
@@ -28,3 +28,9 @@ export function filterAvailableAndCompletedForms(
 export const areFormsEqual = (a: Form, b: Form): boolean => {
   return a !== null && b !== null && a.uuid === b.uuid;
 };
+
+export const filterFormsByName = (formName: string, forms: Array<Form>) => {
+  return forms.filter(
+    form => form.name.toLowerCase().search(formName.toLowerCase()) !== -1
+  );
+};
```

Comment:
Why not just include this function in the component? It's only used in that one place. This just adds a layer of indirection.